sources/install.sh

```diff
@@ -1639,6 +1639,12 @@ function install_scout() {
   add-test-command "scout --help"
 }
 
+function install_azurecli(){
+  colorecho "Installing Azure cli"
+  fapt azure-cli
+  add-test-command "az --help"
+}
```

Comment:
Could you add `add-history` if you have some example commands in mind?

Reply:
I am not used to this tool, so I add only one command. The one that is used in the Twitter thread and in the article to detect multi-tenant authentication.
`az ad sp show --id $TENANT_ID`